src/main/java/com/google/api/codegen/transformer/php/PhpPackageMetadataTransformer.java

```diff
@@ -0,0 +1,70 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer.php;
+
+import com.google.api.codegen.InterfaceView;
+import com.google.api.codegen.config.ApiConfig;
+import com.google.api.codegen.transformer.ModelToViewTransformer;
+import com.google.api.codegen.transformer.PackageMetadataNamer;
+import com.google.api.codegen.viewmodel.ViewModel;
+import com.google.api.codegen.viewmodel.metadata.PackageMetadataView;
+import com.google.api.tools.framework.model.Interface;
+import com.google.api.tools.framework.model.Model;
+import com.google.common.collect.Iterables;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/** Responsible for producing package metadata related views for PHP */
+public class PhpPackageMetadataTransformer implements ModelToViewTransformer {
+  private static final String PACKAGE_FILE = "php/composer.snip";
+
+  // TODO: Retrieve the following values from static file
+  // Github issue: https://github.com/googleapis/toolkit/issues/848
```

Comment:
FYI. This functionality has been implemented in: This https://github.com/googleapis/toolkit/pull/870 which is still pending. I can address this in a separate PR to fix this after Jacob's PR is merged.